modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -417,9 +423,9 @@ cv::GCompiled cv::gimpl::GCompiler::produceCompiled(GPtr &&pg)
 
 cv::GStreamingCompiled cv::gimpl::GCompiler::produceStreamingCompiled(GPtr &&pg)
 {
-    const auto &outMetas = GModel::ConstGraph(*pg).metadata()
-        .get<OutputMeta>().outMeta;
-    std::unique_ptr<GStreamingExecutor> pE(new GStreamingExecutor(std::move(pg)));
+    std::unique_ptr<GStreamingExecutor> pE(new GStreamingExecutor(std::move(pg), m_args, m_metas));
+
+    GMetaArgs outMetas{};
```

Comment:
What's the reason to have empty `outMetas` here? if `compileStreaming()` has been called WITH meta (hope this case is still supported), then the above code is perfectly handled.

You need to handle both cases here.

Reply:
Now outMetas is 
`const auto &outMetas = GModel::ConstGraph(*pg).metadata().get<OutputMeta>().outMeta;`